### PR TITLE
Various cleanup for the kubeconform command

### PR DESCRIPTION
**Pull request by johnbelamaric**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Fix some small issues and documentation with the `kubeconform` command.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
This is a tooling change only.

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: test/conformance/kubeconform/link.go [hunk 1]
```diff
@@ -45,23 +45,30 @@ func link(o *options) {
...
 		})
 	if err != nil {
-		fmt.Printf("%v", err)
-		return
+		return err
+	}
+	fmt.Println()
+	fmt.Printf("Using behaviors from these %d files:\n", len(behaviorFiles))
+	for _, f := range behaviorFiles {
+		fmt.Println("    ", f)
```
Comment (Jefftree) [inline test/conformance/kubeconform/link.go:53]: Is the space just for readability?

File: test/conformance/kubeconform/link.go [hunk 2]
```diff
@@ -28,15 +28,15 @@ import (
 	"k8s.io/kubernetes/test/conformance/behaviors"
 )
 
-func link(o *options) {
+func link(o *options) error {
```
Comment (Jefftree) [inline test/conformance/kubeconform/link.go:31]: Returning the error makes the code much cleaner. Thanks!

**Review by Jefftree — COMMENTED** [review]
Thanks John!

/lgtm
/hold (for additional reviews if necessary)

File: test/conformance/kubeconform/link.go (hunk 1 above)
Comment (johnbelamaric) [reply test/conformance/kubeconform/link.go:53]: yes. though if primary consumer ends up being machines we'll want to tweak this output

**Comment by johnbelamaric** [comment]
@Jefftree thanks. don't know why that command is different from all the others which do "remove-<label>".

**Pull request merged**